samples/Magick.NET.Samples/Drawing.cs

```diff
@@ -35,5 +37,36 @@ public static void DrawText()
                   .Draw(image);
             }
         }
+
+        public static void AddTextToExistingImage()
+        {
+            var pathToBackgroundImage = Path.Combine(Environment.CurrentDirectory, "Files/2FD-Background.jpg");
```

Comment:
Could you use the `Files` class? And add the images to the repo?

Reply:
👍